homeassistant/components/lyric/climate.py

```diff
@@ -390,3 +407,12 @@ async def async_set_hold_time(self, time_period: str) -> None:
         except LYRIC_EXCEPTIONS as exception:
             _LOGGER.error(exception)
         await self.coordinator.async_refresh()
+
+    async def async_set_fan_mode(self, fan_mode: str) -> None:
+        """Set fan (On, Auto, Circulate) mode."""
```

Comment:
Need to translate from HA's FAN_DIFFUSE to Resideo API's "Circulate"

Reply:
There is a couple of predefined modes. If possible then use these. 
https://developers.home-assistant.io/docs/core/entity/climate/

Reply:
Using predefined fan modes will enable translations.

Reply:
mapped